dataflow/gpu-workers/requirements.txt

```diff
@@ -0,0 +1,4 @@
+Pillow==8.2.0
+apache-beam[gcp]==2.29.0
```

Comment:
thanks for restoring the example. nit: 2.30.0 here as well

Reply:
Good catch! Thanks